### ISSUE TITLE
Install problem(s)

### ISSUE THREAD

**Issue by bobbycircle** [coreybutler/nvm-windows#23]
Ok, I'm having a lot of issues, but I would LOVE to get this working on my windows desktop instead of my older mac laptop. Thanks very much in advance for creating nvm for windows! I'm using Win7 64-bit.

The installer puts the files in my username\AppData\Roaming folder by default. Is this correct behavior? If these files follow me to another windows computer, will everything still work?

When cmd.exe is run at the end of the installer, it complains about a missing \settings.txt, even though it is present. If I close that cmd.exe and start a new one, the error message goes away. (Possibly the cmd.exe launched from the installer has not yet picked up the new %path% environment variable?)

Even though the '\settings.txt' error message has gone away from subsequent cmd.exe invocations, the path is still not set up properly-- %NVM_HOME% and %NVM_SYMLINK% are listed when I inspect the path from the command line. (I mentioned this in another issue thread. Workaround is to add the proper dirs to the path manually a la "set path=%path%;%nvm_home%;%nvm_symlink%").

When I open up Win7's 'Environment Variables' dialog box, it looks like the two nvm-windows variables are both in my User Variables as well as my System Variables. I'm not used to seeing variables duplicated like that.

Running "nvm use [...]" causes two separate windows security dialogs to open due to making changes to the symlink in \Program Files.

I run "nvm install 0.10.33" and it seems to work, but only after I run "nvm use ..." which is a little odd since this is the first and only node.js I've installed thus far. But maybe that's how the 'nix versions work.

When I then install another node.js, 0.11.14, install succeeds. When I "run nvm use 0.11.14" that works too. And when I run "nvm list" I can see my two installed versions, although I'm seeing "?-bit executable" instead of "64-bit".

When I try and switch from 0.11.14 to 0.10.33, I get the error "node v0.10.33 (64-bit) is not installed." I get a similar error when I run "nvm use 0.11.14".

So then I run the uninstaller for nvm-windows in the control panel. For some reason a new PATH variable is now added to my User variables where none was there before. It appears to be a copy of my System variable PATH. So now when I open a cmd.exe, my PATH is doubled in size and content. (Deleting my User variable for PATH fixes this.)

**Comment by dscmbobulated** [comment]
@colmsjo Thanks!  Removing the user vars, fixed my issues.

@coreybutler (I was seeing the duplicates.)

**Comment by mdeggies** [comment]
Thanks, @colmsjo! This has taken me awhile to figure out. I also needed to add a NODE_PATH (set to C:\Users\{USER}\AppData\Roaming\npm), which I didn't have previously.

**Comment by absynce** [comment]
@colmsjo I had to remove duplicate `NVM_HOME` and `NVM_SYMLINK` user variables as well, but I couldn't find much of an answer to why outside of an acknowledgement in a [comment on one of the answers](http://superuser.com/questions/27826/why-cant-windows-handle-an-environment-variable-in-path#comment1101483_348544).

**Comment by coreybutler** [comment]
Stale. If this issue crops up again, please open a new issue.
